networks/injective/codegen-injective/index.ts

```diff
@@ -0,0 +1 @@
+export * from './injective/client'
```

Comment:
Does this whole codegen folder for injective have to be checked into main?

Reply:
Good idea, I've deleted the folder and add it to gitignore